logos-codegen/src/parser/mod.rs

```diff
@@ -253,22 +238,16 @@ impl Parser {
                         );
                     }
                 }),
-                ("source", |parser, span, value| match value {
```

Comment:
The removal of `"source"` and the addition of `"utf-8"` option should be documented in the book.